Remove Octave references (#2686)
GMLC-TDC/HELICS: docs/user-guide/installation/language.md

## HELICS with MATLAB

> The Matlab interface to HELICS is found at [matHELICS](https://github.com/gmlc-tdc/mathelics) repository.

Reviewer: ```suggestion
The Matlab interface to HELICS is found in the [matHELICS](https://github.com/gmlc-tdc/mathelics) repository.
```

## HELICS with Simulink

> A simulink interface to HELICS is found at [simHELICS](https://github.com/gmlc-tdc/simhelics) repository. This is a minimalistic interface that is build using simulink S functions.

Reviewer: ```suggestion
A Simulink interface to HELICS is found in the [simHELICS](https://github.com/gmlc-tdc/simhelics) repository. This is a minimalistic interface that is built using Simulink S-functions.
```

## HELICS with Octave

> The Octave interface to HELICS is found at [matHELICS](https://github.com/gmlc-tdc/mathelics) repository. It is currently built with the Matlab Interface

Reviewer: ```suggestion
The Octave interface to HELICS is found in the [matHELICS](https://github.com/gmlc-tdc/mathelics) repository. It is currently built with the Matlab Interface.
```
